scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/configuration/ExecutorEnvironmentalVariables.java

```diff
@@ -67,6 +68,7 @@ private void populateEnvMap(Configuration configuration) {
         if (configuration.isFrameworkUseDocker()) {
             addToList(native_mesos_library_key, native_mesos_library_path);
         }
+        addToList(ES_JAVA_OPTS, getHeapSpaceString(configuration));
```

Comment:
Errr, no, sorry, this is really wrong.

ES_JAVA_OPTS="192m" is blatantly wrong.

You should be aiming for: "-Xms192m -Xmx192m".
Treat the space string as part of what you set